CONTRIBUTING.md

```diff
@@ -19,6 +19,7 @@ To build TiKV you'll need to at least have the following installed:
 * `make` - Build tool (run common workflows)
 * `cmake` - Build tool (required for gRPC)
 * `awk` - Pattern scanning/processing language
+* [`protoc`](https://github.com/protocolbuffers/protobuf/releases) - Google protocol buffer compiler
```

Comment:
We have drop the dependency long time ago, I believe it's introduced back because of etcd-client. /cc @YuJuncen

Reply:
Yes, more preciously it is `prost` that brings this requirement. After PD exposing some of the operations over the etcd key space in `pdpb`, I guess we would be able to remove the requirement of etcd soon.